### DISCUSSION TITLE
Everyday Usage

### DISCUSSION THREAD

**Discussion by VJZV** [Homebrew/discussions#4364] [Everyday usage]
### Output of `brew config`

```shell
brew install brewsci/bio/prokka

Error: prokka: Failed to download resource "tbl2asn--doc"
Download failed: https://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/DOCUMENTATION/tbl2asn.txt
```

### Output of `brew doctor`

```shell
% brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!
```

### Description of issue

Hi all! 
I hope someone can help me with this. I was trying to install prokka using homebrew. However, I keep getting the same error while trying to download "tbl2asn--doc". I would really appreciate any ideas on how to solve the issue. 

Many thanks!
VZ

**Comment by SMillerDev** [comment]
Issues for brewsci/bio can be filed at https://github.com/brewsci/homebrew-bio/issues
